qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -110,20 +110,17 @@ def _run_job(self, job_id, qobj, backend_options, noise_model, validate):
         if validate:
             validate_qobj_against_schema(qobj)
             self._validate(qobj, backend_options, noise_model)
-        qobj_str = self._format_qobj_str(qobj, backend_options, noise_model)
-        output = json.loads(self._controller(qobj_str).decode('UTF-8'))
+        output = self._controller(self._format_qobj(qobj, backend_options, noise_model))
         self._validate_controller_output(output)
         end = time.time()
         return self._format_results(job_id, output, end - start)
 
-    def _format_qobj_str(self, qobj, backend_options, noise_model):
+    def _format_qobj(self, qobj, backend_options, noise_model):
```

Comment:
Why changing the name? What's the link with integrating `pybind11`?

Reply:
_format_qobj_str previously created a dict (.to_dict) and then converted that dict to a json object and returned a string (json.dumps) representation - the new implementation returns a python object (dict) which is then sent directly through to the c++ wrappers - that python object is then converted directly to the binary c++ json object (thus skipping the serialization and 'stringification' on the python side)
TLDR and to respond directly to the question, the function name was changed because _format_qobj_str no longer returns a string